src/goto-symex/field_sensitivity.cpp

```diff
@@ -270,9 +270,19 @@ void field_sensitivityt::field_assignments_rec(
   }
 }
 
-bool field_sensitivityt::is_divisible(
-  goto_symex_statet &state,
-  const ssa_exprt &expr) const
+bool field_sensitivityt::is_divisible(const ssa_exprt &expr)
 {
-  return expr != get_fields(state, expr);
+  if(expr.type().id() == ID_struct || expr.type().id() == ID_struct_tag)
+    return true;
```

Comment:
This change should be squashed with the first "field sensitivity" commit

Reply:
Possible, but I'd rather maintain proper attribution (@smowton has contributed these improvements).